src/styles/main.scss

```
@@ -0,0 +1,91 @@
+$center-size: 30px;
+$clock-size: 500px;
+
+$seconds-hand-width: 10px;
+$seconds-hand-heigh: 225px;
+
+$minute-hand-heigh: 150px;
+$minute-hand-width: 10px;
+
+body {
+  margin: 0;
+  padding: 0;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 100vh;
+}
+
+.clock {
+  animation-iteration-count: infinite;
```

```suggestion
  animation-iteration-count: infinite;
```
i think it's redudant here)